configs/_base_/default_runtime.py

```diff
@@ -8,6 +8,11 @@
     checkpoint=dict(type='CheckpointHook', interval=10),
     sampler_seed=dict(type='DistSamplerSeedHook'),
     visualization=dict(type='PoseVisualizationHook', enable=False),
+    badcase=dict(type="BadCaseAnalyzeHook", 
+                 metric_type="loss",
+                 badcase_thr=100,
+                 show=True,
+                 out_dir='badcase')
```

Comment:
```suggestion
    badcase=dict(type="BadCaseAnalyzeHook", 
                 enable=False,
                 metric_type="loss",
                 badcase_thr=100,
                 show=True,
                 out_dir='badcase')
```

Reply:
Should we set `enable=False` by defauts?

Reply:
Yes, I forgot this.

Reply:
Is it necessary since the default `enable` is `False` in `__init__` method?